File: Program/CPP/Film.cpp
#include <iostream>
#include <string>

using namespace std;

class Film {
private:
    string id;
    string judul;
    string genre;
    int durasi;     // Durasi film dalam menit
    double rating;   // Rating film (contoh: 8.5)

public:
    // Constructor kosong
    Film() {}

    // Constructor berparameter
    Film(string id, string judul, string genre, int durasi, double rating) {
        this->id = id;
        this->judul = judul;
        this->genre = genre;
        this->durasi = durasi;
        this->rating = rating;
    }

    // Destruktor
    ~Film() {}

    // Getter
    string getId() { return id; }
    string getJudul() { return judul; }
    string getGenre() { return genre; }
    int getDurasi() { return durasi; }
    double getRating() { return rating; }

    // Setter
    void setId(string id) { this->id = id; }
    void setJudul(string judul) { this->judul = judul; }
    void setGenre(string genre) { this->genre = genre; }
    void setDurasi(int durasi) { this->durasi = durasi; }
    void setRating(double rating) { this->rating = rating; }
};

File: Program/Java/Film.java
public class Film {
    private String id;
    private String judul;
    private String genre;
    private int durasi;
    private double rating;

    // Constructor kosong
    public Film() {}

    // Constructor berparameter
    public Film(String id, String judul, String genre, int durasi, double rating) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.durasi = durasi;
        this.rating = rating;
    }

    // Getter dan Setter
    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getJudul() {
        return judul;
    }

    public void setJudul(String judul) {
        this.judul = judul;
    }

    public String getGenre() {
        return genre;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }

    public int getDurasi() {
        return durasi;
    }

    public void setDurasi(int durasi) {
        this.durasi = durasi;
    }

    public double getRating() {
        return rating;
    }

    public void setRating(double rating) {
        this.rating = rating;
    }
}